### ISSUE TITLE
Move the probedb.py script

### ISSUE THREAD

**Issue by ksuderman** [galaxyproject/galaxy-helm#467]
The `probedb.py` script that is used for the `livenessProbe` (and others) lives in the `/tmp` directory, so eventually the tmp cleanup job to be added in #448 will break the probes.  Move the script somewhere like `/usr/local/bin/` before #448 is merged.

**Comment by nuwang** [comment]
The mapped file should be read-only and not deletable I believe, so I don't think the cleanup screen will affect it, but if there's a better location we can move it to, that would still make sense.

**Comment by ksuderman** [comment]
I just checked on a cluster I just launched and the `probedb.py` script is `rw` for root and the cron jobs run as root by default. We could set the mode to `444` or `400` or something similar, but that might just lead to unexpected surprises in the future.  I'll get a PR for this ASAP since the maintenance PR was merged.

**Comment by ksuderman** [comment]
Good point. Since it is a config map and not an actual file it shouldn't be deletable.  I'll test, but I would feel better if the script didn't live in `/tmp`

**Comment by ksuderman** [comment]
This is a bit of a false alarm.  The cronjob cleans up `/galaxy/server/database/tmp` and not the `/tmp` directory where the probedb script lives.  It might be a best practice to move it somewhere like `/usr/local/bin`, but it is not urgent.
